drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml

```diff
@@ -86,7 +86,6 @@
   <inherits name="org.kie.workbench.common.dmn.project.DMNProjectClient"/>
 
   <!-- Social Home page -->
```

Comment:
Can't we remove the whole section?

Reply:
@kkufova I asked about this and this bit is still needed. I think there is dead code, but some bits are used.

Reply:
@Rikkola, alright!

Reply:
@Rikkola, alright!